spicy/src/rt/driver.cc

```diff
@@ -51,7 +51,7 @@ void Driver::_debug_stats(const hilti::rt::ValueReference<hilti::rt::Stream>& da
     debug(fmt("input : size-current=%s size-total=%s chunks-cur=%s offset-head=%" PRIu64 " offset-tail=%" PRIu64,
               data_size_cur, data_size_total, data_chunks, data_begin, data_end));
 
-    auto stats = hilti::rt::memory_statistics();
+    auto stats = hilti::rt::resource_usage();
```

Comment:
It would probably make sense to output the added information on l.60.

This would be simplest and easiest to maintain if we'd provide an `operator<<` for `ResourceUsage` and then just output `debug(fmt("%s", &stats))`.

Reply:
I was debating about that as well but ended up leaving as it is because I'm not sure that information would be very helpful in that debugging output.